scripts/format.sh

```diff
@@ -26,6 +29,12 @@ do
     fi
 done
 
+# Automatically upgrade packages to newest versions
+# Assumes uv is installed, if not run pip install uv
+if [ "$SKIP_UPGRADE" = false ]; then
+    uv pip install --no-deps --upgrade ruff yamlfix
```

Comment:
```suggestion
    pip install uv
    uv pip install --no-deps --upgrade ruff yamlfix
```

Reply:
@avishniakov in that case, we can drop line 33, can't we?

Reply:
Yep, sure thing!
The reason why I think it would be better to add this install line is the fact that `uv` might be not so popular around yet, so people can get confused about why it is needed and what it is actually. Having the disabler flag at hand and not so drastic a runtime impact, I'd prefer to make it easier to use, even if it is a bit slower. Normally people are not running `format.sh` on every save, like I do, I guess 🙂

Reply:
Good point. Let me commit your suggestion then drop the line with an additional commit :)